x-pack/plugins/cases/server/client/cases/bulk_update.ts

```diff
@@ -193,6 +193,12 @@ async function getAlertComments({
   });
 }
 
+function haveSameElements(arr1?: Array<{ uid: string }>, arr2?: Array<{ uid: string }>): boolean {
+  if (!arr1 || !arr2 || arr1.length !== arr2.length) return false;
+  const set1 = new Set(arr1);
+  return arr2.every((item) => set1.has(item));
```

Comment:
How does this work as they would be different obj refs, unless I missed something? I wish we had access to [difference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/difference)